tests/syrupy.py

```diff
@@ -111,10 +110,6 @@ def _serialize(
             serializable_data = cls._serializable_config_entry(data)
         elif dataclasses.is_dataclass(data):
             serializable_data = dataclasses.asdict(data)
-        elif isinstance(data, IntFlag):
-            # The repr of an enum.IntFlag has changed between Python 3.10 and 3.11
-            # so we normalize it here.
-            serializable_data = _IntFlagWrapper(data)
```

Comment:
Unsure about this one, but relates to #90223

Reply:
I think it's better to not include things in the PR that you aren't sure about. That way the PR isn't blocked until someone that knows comes around.

Reply:
Fair enough, dropped this change.